testBaseData/numericNames/database_config.yaml

```diff
@@ -0,0 +1,31 @@
+schema:
+  instanceName: Test
+  metadata:
+    - name: date
+      type: date
+    - name: dateSubmitted
+      type: date
+    - name: "2"
+      type: string
+    - name: age
+      type: int
+    - name: sex
+      type: string
+    - name: pangoLineage
+      type: pango_lineage
+    - name: qc
+      type: float
+    - name: accession
+      type: string
+    - name: version
+      type: int
+    - name: submissionId
+      type: string
+    - name: accessionVersion
+      type: string
+    - name: isRevocation
+      type: string
+    - name: versionStatus
+      type: string
```

Comment:
Are all those fields important for the test case? If not, I would suggest to remove them (only keeping the primary key).